snippets/csharp/System/NullReferenceException/Overview/Array1.cs

```diff
@@ -1,16 +1,16 @@
 ﻿// <Snippet8>
 using System;
 
-public class Example
+public class Array1Example
 {
-   public static void Main()
-   {
-      String[] values = { "one", null, "two" };
-      for (int ctr = 0; ctr <= values.GetUpperBound(0); ctr++)
-         Console.Write("{0}{1}", values[ctr].Trim(),
-                       ctr == values.GetUpperBound(0) ? "" : ", ");
-      Console.WriteLine();
-   }
+    public static void Main()
+    {
+        string[] values = { "one", null, "two" };
```

Comment:
We could use collection expressions here:

```suggestion
        string[] values = [ "one", null, "two" ];
```